api/http/server.go

```diff
@@ -897,6 +907,66 @@ func (s *Server) HandleGetFile(w http.ResponseWriter, r *http.Request) {
 	http.ServeContent(w, r, fileName, time.Now(), newBufferedReadSeeker(reader, getFileBufferSize))
 }
 
+// HandleGetTag responds to the following request
+//    - bzz-tag:/<manifest>  and
+//    - bzz-tag:/?tagId=<tagId>
+// Clients should use root hash or the tagID to get the tag counters
+func (s *Server) HandleGetTag(w http.ResponseWriter, r *http.Request) {
+	getTagCount.Inc(1)
+	uri := GetURI(r.Context())
+	if uri == nil {
+		getTagFail.Inc(1)
+		respondError(w, r, "Error decoding uri", http.StatusBadRequest)
+		return
+	}
+	fileAddr := uri.Address()
+
+	var tag *chunk.Tag
+	if fileAddr == nil {
+		tagString := r.URL.Query().Get("tagId")
```

Comment:
Maybe to rename `tagId` to `id`? This is already on bzz-tag scheme, so tag is implicit.

Reply:
Done